File: Programmers/가장 큰 수.cpp
// level 2
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

bool compare(string a, string b) {
   
    return a + b > b + a;
}

string solution(vector<int> numbers) {
    string answer = "";
    vector<string> v;
    
    for (int number : numbers) {
        v.push_back(to_string(number));
    }
    
    sort(v.begin(), v.end(), compare);
    
    for (string vv: v) {
        answer += vv;
    }
    if (answer[0] == '0')
        answer = "0";
    return answer;
}

File: Programmers/가장 큰 수.py
# level 2
def solution(numbers):
    answer = ''
    str_num = []
    
    for num in numbers:
        str_num.append(str(num))
    
    str_num.sort(key = lambda n : n * 3, reverse = True)
    
    for s in str_num:
        answer += s
    
    if answer[0] == '0':
        answer = '0'
    return answer